src/renderer/wg_engine/tvgWgRenderer.cpp

```diff
@@ -273,9 +274,13 @@ bool WgRenderer::sync() {
             WGPURenderPassColorAttachment colorAttachment{};
             colorAttachment.view = backBufferView;
             colorAttachment.resolveTarget = nullptr;
-            colorAttachment.loadOp = WGPULoadOp_Clear;
+            if (mClearBuffer) {
```

Comment:
@SergeyLebedkin Please confirm

Reply:
colorAttachment.clearValue = { 0.0, 0.0, 0.0, 1.0 };
move floats to doubles
alpha channel should be 1.0, i guess

Reply:
I intentionally changed it. the cleared buffer should become total transparent, should not have any alpha values. So the previous is inccorect.

Reply:
OK, And by the spec color channels values have a double type